flocker/acceptance/scripts/lsblkhttp.py

```diff
@@ -0,0 +1,36 @@
+"""
+HTTP server that returns the size of the mounted volume.
+"""
+
+from subprocess import check_output
+
+from BaseHTTPServer import BaseHTTPRequestHandler, HTTPServer
+
+
+class Handler(BaseHTTPRequestHandler):
+    def do_GET(s):
+        s.send_response(200)
+        s.end_headers()
+        if "is_http" in s.path:
+            s.wfile.write(b"hi")
+        else:
+            try:
+                mounts = {}
+                for line in check_output(['mount', '-l']).split('\n'):
+                    parts = line.split(' ')
+                    if len(parts) > 2:
+                        mounts[parts[2]] = parts[0]
+                device_path = mounts["/sizedvoldata"]
+
+                command = [b"/bin/lsblk", b"--noheadings", b"--bytes",
+                           b"--output", b"SIZE", device_path.encode("ascii")]
```

Comment:
agreed, I think the issue was i took code form another script we had as an example, and found the mount output example and never matched up how to treated encoding. Will make it consistent.